handler/src/main/java/io/netty/handler/ssl/JdkSslClientContext.java

```diff
@@ -239,13 +241,13 @@ public JdkSslClientContext(File trustCertChainFile, TrustManagerFactory trustMan
      * @deprecated use {@link SslContextBuilder}
      */
     @Deprecated
-    public JdkSslClientContext(File trustCertChainFile, TrustManagerFactory trustManagerFactory,
+    public JdkSslClientContext(File trustCertCollectionFile, TrustManagerFactory trustManagerFactory,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Constructor has 11 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS00107)